CSharp/Library/QnAMaker/QnAMaker/QnAMakerService/QnAMakerService.cs

```diff
@@ -89,7 +89,7 @@ public sealed class QnAMakerService : IQnAService
         /// <summary>
         /// The base URI for accessing QnA Service.
         /// </summary>
-        public static readonly Uri UriBase = new Uri("https://westus.api.cognitive.microsoft.com/qnamaker/v2.0/knowledgebases");
+        public static readonly Uri UriBaseV2 = new Uri("https://westus.api.cognitive.microsoft.com/qnamaker/v2.0/knowledgebases");
```

Comment:
Breaking change.

Reply:
Not a breaking change. This is internal to the QnAService, to set the right uri depending on the inputs.

Reply:
Added hostname as the last optional parameter, so the interface remains the same for older users.

Reply:
This is a public constant which makes renaming it a breaking change since code could have referred to the old one.  Agreed it is relatively low risk so if you really want to leave it you can, but I don't see the benefit.